### ISSUE TITLE
Search Encrypt malware concerns

### ISSUE THREAD

**Issue by mimmiepower** [CHEF-KOCH/Warez#10]
Search Encrypt (https://www.searchencrypt.com/) is in the list. There have been some [concerns about malware](https://www.reddit.com/r/firefox/comments/aw3sza/is_searchencrypt_legit/) regarding this site. The owners [deny](https://www.searchencrypt.com/truth/) such claims. Personally I do not believe their excuse. If their browser extension is clean, why is not on addons.mozilla.org? Why would someone other than the Search Encrypt themselves hijack user's browsers seemngly (I have not tried it myself) routing traffic through searchencrypt.com?

**Comment by CHEF-KOCH** [comment]
If I'm not mistaken SearchEncrypt 
* is closed source 
* shows ads (_which explains the "Search Encrypt earns revenue from the advertisements that appear above the search results" line_)
* According to a {user vote](https://strawpoll.com/akpdxpc3) (-still running_) no one likes SearchEncrypt. 0 votes.
I think the browser hijacker accusation comes from this [page](https://www.enigmasoftware.com/searchencryptcom-removal/)
* The user itself who posted and started it [doesn't exist anymore](https://addons.mozilla.org/en-US/firefox/user/12054389/).

An alternative could be https://searx.pwoss.org/ which is FOSS and hosted by "cyber-sec" experts.

**Comment by maximousblk** [comment]
Any update on this?

**Comment by CHEF-KOCH** [comment]
Can you provide any evidence to confirm or deny the claims?!

I found nothing, that's why the ticket isn't closed.

**Comment by maximousblk** [comment]
Does this help?
![image](https://user-images.githubusercontent.com/28438021/80313909-b3284500-880b-11ea-815a-ef1861be30d7.png)

**Comment by CHEF-KOCH** [comment]
No, depending on the uBlock "blocking mode" the results are different, it also depends on the filter lists you use. 

However, uBlock is not a tool to provide evidence if there is malware involved or not, at best it's an indicator (remember that false/positives are also possible in filter lists). 

Professional investigation requires more than just measurements by specific tools/extensions. I did checked:
- Their Tos
- Domain and reputation
- Privacy Policy
-I even contacted them and they deny the claim
- Inspected and audited the Browser extension (the claim is that the extension delivers or even contains malware NOT the domain btw, so uBlock is useless as measurement tool anyway).
- Checked if traffic (once the extension is installed) gets redirected or altered, nothing. The Browser itself also cannot be hijacked via an extension (not on Chrome nor Fiefox) without specific permissions, if you check the permissions, they are pretty much okay.

However I took the accusations very serious and unless someone provides evidence that Search Encrypt really delivers malware "somehow" I leave the ticket open. If no one proved something within the next 2 days I close it and mark at as resolved because I think that the service is pretty much good to use.
